packagedoc/additional_docs/History.tex

```diff
@@ -86,4 +86,7 @@
 - Prevented side effects of token string in error messages log\newline
 - Checked absolute path when overwriting parameter}
 
+\historyversiondate{0.8.2}{10/2024}
+\historychange{- Enhanced import JSON file feature which allows dynamic path of imported file}
+
```

Comment:
Hi Son,

please rephrase to:

_Enabled dynamic paths for imported JSON files (based on dollar operator expressions)_